### ISSUE TITLE
Additional image features

### ISSUE THREAD

**Issue by MaxNoe** [cta-observatory/cta-lstchain#346]
There are several image features in ctapipe not currently calculated in the lst chain.

This includes:

- [ ] conctentrations
- [ ] leakage2 and leakages expressed in terms of pixels not amount of light
- [ ] General descriptive statistics of the light distribution and pulse times (min, mean, max, std)

**Comment by MaxNoe** [comment]
Concetration and the other leakage values are already available in ctpaipe 0.7.

Should I add them now or do you want to wait with reprocessing until 0.8?

**Comment by vuillaut** [comment]
> Concetration and the other leakage values are already available in ctpaipe 0.7.
> 
> Should I add them now or do you want to wait with reprocessing until 0.8?

I agree they should be computed and added and this can easily be done already.
If you want, please do so already.

**Comment by MaxNoe** [comment]
Ok, will do

**Comment by MaxNoe** [comment]
See #348
